Khan/perseus
# Use getSaveWarnings to stop 0 length locked line from saving

## Summary:
Right now, setting the locked line's points to be the same
will show an error, but it won't stop the exercise from
being able to be saved.

By using `getSaveWarnings` here, we can do that.

This PR includes:
- Storybook story that shows the save warnings on the side
- Util function for checking locked points + tests
- Util function being used in `getSaveWarnings()` as well as inside locked line settings

Issue: https://khanacademy.atlassian.net/browse/LEMS-2008

## Test plan:
Storybook 
- http://localhost:6006/?path=/story/perseuseditor-editorpage--with-save-warnings
- Set the points to be the same in the locked line settings
- Make sure that a save warning shows up in the save warning box

<img width="1494" alt="Screenshot 2024-05-21 at 5 14 05 PM" src="https://github.com/Khan/perseus/assets/13231763/89103fdb-3647-45f4-b6c1-8a0f9af90af2">
